test/compose.js

```diff
@@ -65,4 +65,14 @@ describe('compose', function() {
     assert.deepEqual(g(1, 2, 3), [1, 2, 3]);
   });
 
+  it('returns a curried function', function() {
+    function f(a) { return a + '_END'; }
+    function g(b, c, d) { return [b, c, d].join('_'); }
+    var c = R.compose(f, g);
```

Comment:
Interesting. I'd thought the alphabetical ordering of `f` and `g` was suggestive of application order, as in this [diagram][1]:

![g ∘ f](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Compfun.svg/600px-Compfun.svg.png)

It sounds strange to me to say _composition of `g` and `f`_, in the same way that it sounds strange to me to say _dad and mum_ rather than _mum and dad_. There's no logical reason for this, of course. :)


[1]: https://en.wikipedia.org/wiki/Function_composition#/media/File:Compfun.svg